app/assets/javascripts/application.js

```diff
@@ -66,10 +66,30 @@ var checkboxSelector = (function() {
   return {
     init: cs.init
   }
-})();
+})()
+
+var yesNoButtons = (function () {
```

Comment:
this is a very cool solution! would be nice to move these JS functions into different files at some point if we can